rocky/tools/templatetags/findings_count.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def total_findings(data: dict[str, Any]) -> int:
+    total_findings = 0
+    for ip in data.values():
+        total_findings += int(ip["summary"]["total_findings"])
+    return total_findings
```

Comment:
```suggestion
   return sum([ int(ip["summary"]["total_findings"]) for ip in data.values()])
```

Reply:
Nice suggestion @Rieven 

An even better approach would be using a generator expression instead. This saves an extra iteration by `sum`, because you don't have to construct an entire list to loop over it again.
```python3
return sum((int(ip["summary"]["total_findings"]) for ip in data.values()))
```